logging/net_appender.go

```diff
@@ -439,7 +439,7 @@ func CreateNewGRPCClient(ctx context.Context, cloudCfg *CloudConfig) (rpc.Client
 		dialOpts = append(dialOpts, rpc.WithInsecure())
 	}
 
-	return rpc.DialDirectGRPC(ctx, grpcURL.Host, NewLogger("netlogger"), dialOpts...)
+	return rpc.DialDirectGRPC(ctx, grpcURL.Host, GetOrNewLogger("rdk.network_outgoing.netlogger"), dialOpts...)
```

Comment:
This logger is guaranteed to exist from the line above, but this was the most straightforward way to retrieve it.